<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="indexLayout" native="true">
   <property name="geometry">
    <rect>
     <x>-400</x>
     <y>-502</y>
     <width>1000</width>
     <height>1400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft New Tai Lue</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#indexLayout {
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(141, 176, 255, 255), stop:1 rgba(255, 236, 202, 255));
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>590</y>
      <width>371</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Lucida Sans Unicode&quot;;
margin-bottom: 50px;</string>
       </property>
       <property name="text">
        <string>Start a new session</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="realBtn">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
	font: 12pt &quot;Lucida Sans Unicode&quot;;
	border-radius: 8px;
	background: rgba(255, 255, 255, 0.3);
	padding: 20px;
}

*:hover {
	background: rgba(255, 255, 255, 0.7);
}</string>
       </property>
       <property name="text">
        <string>Mode Real</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="demoBtn">
       <property name="font">
        <font>
         <family>Lucida Sans Unicode</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
	border-radius: 8px;
	font: 12pt &quot;Lucida Sans Unicode&quot;;
	background: rgba(255, 255, 255, 0.3);
	padding: 20px;
}

*:hover {
	background: rgba(255, 255, 255, 0.7);
}</string>
       </property>
       <property name="text">
        <string>Mode Demo</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="logoutBtn">
         <property name="font">
          <font>
           <family>Lucida Sans Unicode</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">* {
	border-radius: 8px;
	font: 12pt &quot;Lucida Sans Unicode&quot;;
	background:rgba(255, 0, 0, 0.3);
	padding: 10px;
}

*:hover {
	background: rgba(255, 0, 0, 0.6);
}</string>
         </property>
         <property name="text">
          <string>Log Out</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsBtn">
         <property name="font">
          <font>
           <family>Lucida Sans Unicode</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">* {
	border-radius: 8px;
	font: 12pt &quot;Lucida Sans Unicode&quot;;
	background: rgba(255, 255, 255, 0.3);
	padding: 10px;
}

*:hover {
	background: rgba(255, 255, 255, 0.7);
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="font">
          <font>
           <family>Microsoft New Tai Lue</family>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="title">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-2</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>24</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Billeasy</string>
          </property>
         </widget>
         <widget class="QLabel" name="subtitle">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>The Application</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="font">
          <font>
           <family>Microsoft New Tai Lue</family>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="wlcTextLbl">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>41</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Hello,</string>
          </property>
         </widget>
         <widget class="QLabel" name="wlcUsernameLbl">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>0</y>
            <width>201</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>USERNAME</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="walletEdit">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>10</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding: 7px;</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>10</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Wallet Start</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>16</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>€</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
